livekit_internal.proto

```diff
@@ -113,6 +113,9 @@ message RecordingReservation {
   string id = 1;
   int64 submitted_at = 2;
   StartRecordingRequest request = 3;
+  string api_key = 4;
```

Comment:
is it a good idea to send api key/secret over the wire? can it be a token instead?